clang/include/clang/Driver/Options.td

```diff
@@ -961,6 +961,10 @@ def Wdeprecated : Flag<["-"], "Wdeprecated">, Group<W_Group>,
   HelpText<"Enable warnings for deprecated constructs and define __DEPRECATED">;
 def Wno_deprecated : Flag<["-"], "Wno-deprecated">, Group<W_Group>,
   Visibility<[ClangOption, CC1Option]>;
+def Winvalid_constexpr : Flag<["-"], "Winvalid-constexpr">, Group<W_Group>,
```

Comment:
I don't think we want these to be separate.  There is a variety of 'Flags' somewhere that auto generates both.

Reply:
This was a bit more involved than you might expect, but it's done now and I think it's in a better state as a result.